<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Adashb</class>
 <widget class="QMainWindow" name="Adashb">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="StudentName">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>231</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Product Sans</family>
      <pointsize>15</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="Welcome">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>111</width>
      <height>33</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>6</width>
      <height>5</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>9</width>
      <height>10</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Product Sans</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Welcome,</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>511</width>
      <height>991</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(242, 242, 242);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>401</width>
       <height>381</height>
      </rect>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(218, 218, 218);
border-radius: 10px;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLineEdit" name="saddEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>220</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: #ffffff; /* Background color: white */
    color: #000000;            /* Text color: black */
    border: 1px solid #337ab7; /* Border color: same blue as QPushButton */
    border-radius: 4px;        /* Optional: round the corners */
    padding: 4px;              /* Optional: add some padding */
}
</string>
      </property>
      <property name="placeholderText">
       <string>Eg. MTH101</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>200</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Subject</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_1">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="taddEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: #ffffff; /* Background color: white */
    color: #000000;            /* Text color: black */
    border: 1px solid #337ab7; /* Border color: same blue as QPushButton */
    border-radius: 4px;        /* Optional: round the corners */
    padding: 4px;              /* Optional: add some padding */
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>*****@wvsu.edu.ph</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>401</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(55, 122, 183);
border-radius: 8px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>10</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Product Sans Black</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>ADD FACULTY</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="saddButton">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>220</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #337ab7;  /* Button background color (blue) */
    color: #ffffff;             /* Text color (white) */
	border-radius: 3px;
}
QPushButton:hover{
	background-color: #285e8e;
}
</string>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="taddButton">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>130</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #337ab7;  /* Button background color (blue) */
    color: #ffffff;             /* Text color (white) */
	border-radius: 3px;
}
QPushButton:hover{
	background-color: #285e8e;
}
</string>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="viewButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgb(0, 175, 0);  /* Button background color (blue) */
    color: #ffffff;             /* Text color (white) */
	border-radius: 3px;
}
QPushButton:hover{
	background-color: #285e8e;
}
</string>
      </property>
      <property name="text">
       <string>View Curent List</string>
      </property>
     </widget>
     <widget class="Line" name="line_5">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>310</y>
        <width>3</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="taddEdit_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>130</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: #ffffff; /* Background color: white */
    color: #000000;            /* Text color: black */
    border: 1px solid #337ab7; /* Border color: same blue as QPushButton */
    border-radius: 4px;        /* Optional: round the corners */
    padding: 4px;              /* Optional: add some padding */
}
</string>
      </property>
      <property name="placeholderText">
       <string>Eg. Juan Dela Cruz</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="taddEdit_3">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>70</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: #ffffff; /* Background color: white */
    color: #000000;            /* Text color: black */
    border: 1px solid #337ab7; /* Border color: same blue as QPushButton */
    border-radius: 4px;        /* Optional: round the corners */
    padding: 4px;              /* Optional: add some padding */
}
</string>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>50</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>82</x>
        <y>350</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #377ab7;</string>
      </property>
      <property name="text">
       <string>Total No. of Students:</string>
      </property>
     </widget>
     <widget class="QLabel" name="Studentno">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>350</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>1098</string>
      </property>
     </widget>
     <widget class="QPushButton" name="viewButton_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>340</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgb(0, 175, 0);  /* Button background color (blue) */
    color: #ffffff;             /* Text color (white) */
	border-radius: 3px;
}
QPushButton:hover{
	background-color: #285e8e;
}
</string>
      </property>
      <property name="text">
       <string>View </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="Images.qrc">:/new/prefix1/pngegg.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>401</width>
       <height>191</height>
      </rect>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(218, 218, 218);
border-radius: 10px;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="AcadYear_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Product Sans Light</family>
        <pointsize>18</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Second Semester&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="AcadYear">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>361</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Product Sans Light</family>
        <pointsize>18</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Academic Year: 2024-2025&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="StatusBar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Product Sans</family>
        <pointsize>13</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Evaluation Status:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>90</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Product Sans</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: red;</string>
      </property>
      <property name="text">
       <string>ENDED</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>90</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Product Sans</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: green;</string>
      </property>
      <property name="text">
       <string>ONGOING</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>90</y>
        <width>16</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Product Sans</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: BLACK;</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Coolvetica</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Rating Progress:</string>
      </property>
     </widget>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>401</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="backButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>680</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #337ab7;  /* Button background color (blue) */
    color: #ffffff;             /* Text color (white) */
	border-radius: 3px;
}
QPushButton:hover{
	background-color: #285e8e;
}
</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>-30</x>
       <y>0</y>
       <width>461</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(55, 122, 183);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>0</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>1</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="Images.qrc">:/new/prefix1/wvsu_logo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>8</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>WEST VISAYAS STATE UNIVERSITY
MAIN CAMPUS</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>40</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>8</pointsize>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Lapaz, Iloilo CIty</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Product Sans Black</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>DASHBOARD</string>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>118</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>0</y>
        <width>16</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <zorder>label_7</zorder>
     <zorder>label_8</zorder>
     <zorder>label_9</zorder>
     <zorder>line_2</zorder>
     <zorder>line_3</zorder>
     <zorder>label_5</zorder>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>220</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Coolvetica</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar {
    border-radius: 3px;               /* Border radius to match the other elements */
    text-align: center;               /* Center align the text */
    color: rgb(0, 0, 0);                   /* Text color (white) */
    background-color: #d4d4d4;        /* Background color of the QProgressBar (neutral gray) */
}

QProgressBar::chunk {
    background-color: #337ab7;        /* Progress bar color (blue) matching QPushButton */
    border-radius: 3px;               /* Rounded corners for the progress bar */
}


QProgressBar::chunk:hover {
    background-color: #285e8e;        /* Progress bar color on hover */
}
</string>
     </property>
     <property name="value">
      <number>15</number>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>590</y>
       <width>401</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>424</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
